roles/haproxy/templates/haproxy.cfg.j2

```diff
@@ -35,7 +35,11 @@ global
     daemon
 
     # turn on stats unix socket
+{% if develop %}
+    stats socket /var/lib/haproxy/stats mode 600 level admin
+{% else %}
     stats socket /var/lib/haproxy/stats
```

Comment:
This is just haproxy specific, not sure it should be documented (you should not alter functionality you don't understand in the first place :wink: ). The develop only creates a socket that accepts admin commands, which we need to disable health checks on dev, as is done [here][1].

[1]: https://github.com/OpenConext/OpenConext-deploy/blob/development-environment/roles/haproxy/tasks/main.yml#L42